cypress/support/login.ts

```diff
@@ -1,13 +1,11 @@
-import { createClient } from '@supabase/supabase-js';
+import createSupabaseClient from './create-supabase-client';
```

Comment:
Why is it imported explicitly here instead of calling `cy.createSupabaseClient`?

Reply:
It won't work otherwise for some reason

Reply:
Hmm, it worked for me. I removed the import and simply added `cy.` and all of the tests passed

Reply:
Let's keep it at that to avoid any possibilities of test flake

Reply:
I don't see why it would be flaky, but fine